README.md

```diff
@@ -10,4 +10,30 @@ You can run it with the following command, note that it requires docker:
 
 Then you can open [localhost:8080](http://localhost:8080) on your browser and browse the documentation.
 
-To kill the process, just type ctrl + c
\ No newline at end of file
+To kill the process, just type ctrl + c.
+
+## Preview latest kafka document
+
+1. Assume you have [kafka](https://github.com/apache/kafka) and kafka-site folder structure like this:
+
+```shell
+.
+├── kafka
+└── kafka-site
+```
+
+2. Generate document in kafka folder:
+
+```shell
+./gradlew clean siteDocTar
+tar zxvf core/build/distributions/kafka_2.13-$(./gradlew properties | grep version: | awk '{print $NF}' | head -n 1)-site-docs.tgz
+```
+
+3. Running website in kafka-site folder and open [http://localhost:8080/dev/documentation/](http://localhost:8080/dev/documentation/):
+
+```shell
+rm -rf dev
+mkdir dev
+cp -r ../kafka/site-docs/* dev
+./start-preview.sh
+```
```

Comment:
How about this:

3. Copying the generated documents in Kafka repository into kafka-site, and preview them (note that it requires docker):

```shell
# change directory into kafka-site repository
cd KAFKA_SITE_REPO
# copy the generated documents into dev folder
rm -rf dev
mkdir dev
cp -r ../kafka/site-docs/* dev
# preview it
./start-preview.sh
```

Then you can open [http://localhost:8080/dev/documentation/](http://localhost:8080/dev/documentation/) on your browser and browse the generated documentation.